### PR TITLE
* Peer Down: modified reason code with optional TLV field.

### PR DESCRIPTION
**Pull request by paololucente**

### CODE REVIEW

File: draft-ietf-grow-bmp-local-rib.txt [hunk 1]
```diff
@@ -542,11 +542,16 @@ Internet-Draft                BMP Local-RIB               September 2018
...
 
-   Peer down notification SHOULD follow the section 4.9 [RFC7854] reason
-   2.
+   Peer down notification SHOULD use reason code TBD3. Following the
+   reason is data in TLV format. The following peer Down information
+   TLV type is defined:
 
-   The VRF/Table Name informational TLV SHOULD be included if it was in
-   the Peer UP.
+   o  Type = TBD4: VRF/Table Name.  The Information field contains an
```
Comment (TimEvens) [inline draft-ietf-grow-bmp-local-rib.txt]: VRF Table name is **TBD2**.

File: draft-ietf-grow-bmp-local-rib.txt [hunk 2]
```diff
@@ -542,11 +542,16 @@ Internet-Draft                BMP Local-RIB               September 2018
 
 5.3.  Peer Down Notification
 
-   Peer down notification SHOULD follow the section 4.9 [RFC7854] reason
-   2.
+   Peer down notification SHOULD use reason code TBD3. Following the
```
Comment (TimEvens) [inline draft-ietf-grow-bmp-local-rib.txt]: TBD3 needs to be added to IANA considerations as well.

**Review by TimEvens — CHANGES_REQUESTED** [review]
Looks good, but this change should be in the XML file.  The TXT can then be generated.  Normally I update the XML then generate the text.

**Comment by paololucente** [comment]
Thank you @TimEvens for the review and guidance. I should have taken into account all your comments with the second commit.

File: draft-ietf-grow-bmp-local-rib.txt (hunk 2 above)
Comment (TimEvens) [reply draft-ietf-grow-bmp-local-rib.txt]: Looks good

**Review by TimEvens — APPROVED** [review]

**Pull request merged**